docs/legacy_installation.md

```diff
@@ -0,0 +1,56 @@
+### Legacy installation (without Symfony Flex)
+
+1. Require plugin with composer:
+
+    ```bash
+    composer require sylius/admin-order-creation-plugin
+    ```
+
+2. Import configuration:
+
+    ```yaml
+    imports:
+        - { resource: "@SyliusAdminOrderCreationPlugin/Resources/config/app/config.yml" }
+    ```
+
+3. Import routing:
+
+    ```yaml
+    sylius_admin_order_creation:
+        resource: "@SyliusAdminOrderCreationPlugin/Resources/config/app/routing.yml"
+    ```
+
+4. Add plugin class to your `AppKernel`:
+
+    ```php
+    $bundles = [
+        new \FOS\JsRoutingBundle\FOSJsRoutingBundle(),
+        new \Sylius\AdminOrderCreationPlugin\SyliusAdminOrderCreationPlugin(),
+    ];
+    ```
+
+5. Copy Sylius templates overridden in plugin to your templates directory (e.g `templates/bundles/`):
```

Comment:
Pre-Flex installations do not have the required config for `templates/` directory, they should use the old `app/Resources/somethinghereidontrememberanymore`.